api/tasks/http.go

```diff
@@ -43,6 +47,7 @@ func AddTask(w http.ResponseWriter, r *http.Request) {
 		ObjectID:    &taskObj.ID,
 		Description: &desc,
 	}.Insert()); err != nil {
+		util.UserVaultCache.DecrementUsage(user.Username)
```

Comment:
no error checking

Reply:
I didn't care, because we are in the middle of error handling. As we are panicking in the next line anyways, a cleanup does not make sense at all. However, tasks/http.go has a lot pf panic in there which should be replaced by appropriate logging (see comment above). So I added a basic error handling for the change. I'll leave the panic alone as we should handle this in #462